src/storage/mapping/ExtensionBasedMapper.ts

```diff
@@ -1,82 +1,21 @@
 import { promises as fsPromises } from 'fs';
 import * as mime from 'mime-types';
 import type { ResourceIdentifier } from '../../ldp/representation/ResourceIdentifier';
-import { getLoggerFor } from '../../logging/LogUtil';
 import { APPLICATION_OCTET_STREAM, TEXT_TURTLE } from '../../util/ContentTypes';
 import { NotImplementedHttpError } from '../../util/errors/NotImplementedHttpError';
-import {
-  encodeUriPathComponents,
-  ensureTrailingSlash,
-  isContainerIdentifier,
-  joinFilePath,
-  normalizeFilePath,
-  trimTrailingSlashes,
-} from '../../util/PathUtil';
-import type { FileIdentifierMapper, FileIdentifierMapperFactory, ResourceLink } from './FileIdentifierMapper';
-import { getAbsolutePath, getRelativePath, validateRelativePath } from './MapperUtil';
-
-export interface ResourcePath {
-
-  /**
-   * The path of the container.
-   */
-  containerPath: string;
-
-  /**
-   * The document name.
-   */
-  documentName?: string;
-}
-
-/**
- * A mapper that stores the content-type of resources in the file path extension.
```

Comment:
What is the reason for removing this entire block of comments? I think it might be useful for someone who has no idea why a mapper gets used?

Reply:
Moved it to "base".